lib/lita/handlers/reviewme.rb

```diff
@@ -96,6 +96,7 @@ def comment_on_github(response, room: get_room(response))
         rescue Octokit::Error
           response.reply("Unable to check who issued the pull request. Sorry if you end up being assigned your own PR!")
         end
+        return response.reply('Sorry, no reviewers found') unless reviewer
```

Comment:
Hi @iamvery! Short answer: To be as unobtrusive as possible to current functionality.

Long answer: The other option would be to do a major refactor of this section. To me, seeing a `begin..end` is always an opportunity for a private method and better separation of concerns.  There are multiple times that `reviewer` is set, and well, I didn't want to do any major refactoring.  From my testing, it seems to be working properly and I just wanted to fix the bug. 👍